### ISSUE TITLE
Loss magnitudes

### ISSUE THREAD

**Issue by aakash-saboo** [dongliangcao/Spectral-Meets-Spatial#2]
@dongliangcao 
Hi. I am training the model on a different dataset of cortical surfaces. The magnitudes of the losses are very high, of the order of 10^8. 

<img width="1838" height="40" alt="Image" src="https://github.com/user-attachments/assets/af704d81-e20c-4077-b113-9b2dd16cc4c6" />

I am overfitting on one data point. Preprocessing has been performed using the script, and the weights of the losses are the same. The meshes are rigidly aligned as well
Can you suggest potential problems?
